README.md

```diff
@@ -1136,10 +1136,10 @@ sleepRandom(5).then(result => {
   console.log(result);                 // => 202, after 10 sec.
 }).then(() => {
   console.log('immediately after');    // => immediately after
-  throw Error('Irror!');
+  throw Error('Error!');
 }).then(() => {
   console.log('will not be displayed');
-}).catch(x => console.log(x));         // => => Error: Irror!
+}).catch(x => console.log(x));         // => => Error: Error!
```

Comment:
I can't tell if this is intentional/being cute, or just a 🤷 .

It's trivial for me to drop things if they're undesired.

Reply:
Yes, this is intentional as an example.